client/components/date-range/index.js

```diff
@@ -64,10 +63,9 @@ export class DateRange extends Component {
 
 		// Define the date range that is selectable (ie: not disabled)
 		const firstSelectableDate =
-			has( this.props, 'firstSelectableDate' ) &&
-			this.props.moment( this.props.firstSelectableDate );
+			this.props.firstSelectableDate && this.props.moment( this.props.firstSelectableDate );
```

Comment:
🤔  Won't that evaluate to `true` only if `firstSelectableDate` is truthy? While previously, it was evaluating to `true` if the property existed, regardless of the value. Same for `lastSelectableDate` below.

Reply:
That's fine though, because the prop is either not present, or is a valid date that can be passed to the `moment` constructor.

And if you look at other methods in this component, like `isValidDate` or `getDisabledDaysConfig`, they already do the boolean check.

Reply:
Gotcha!